src/createServer.js

```diff
@@ -1,3 +1,39 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { findErrors } = require('./validation');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    res.setHeader('Content-type', 'application/json');
+
+    const { url } = req;
+    const normURL = new URL(url, 'http://localhost:5700');
```

Comment:
`const PORT = process.env.PORT || 5700;`
It's better to do it in an environment variable